src/frontend/src/components/nav/PageDetail.tsx

```diff
@@ -50,12 +54,21 @@ export function PageDetail({
     ]
   ]);
 
+  // breadcrumb caching
+  const computedBreadcrumbs = useMemo(() => {
```

Comment:
@matmair I would suggest that a simpler solution, rather than adding another `last_crumb` parameter, would be to simply remove the last one in the list, if the setting is not enabled?

Reply:
Not all pages have breadcrumbs where it makes sense to remove the last one. I find an explicit interface better than one that assumes things and needs to be negated.

Reply:
Ok, I'm happy with that then